packages/glimmer-runtime/lib/compiled/opcodes/dom.ts

```diff
@@ -386,17 +386,19 @@ export class DynamicAttrNSOpcode extends Opcode {
   public type = "dynamic-attr";
   public name: InternedString;
   public namespace: InternedString;
+  public isTrusting: boolean;
 
-  constructor({ name, namespace }: { name: InternedString, namespace: InternedString }) {
+  constructor({ name, namespace, isTrusting }: { name: InternedString, namespace: InternedString, isTrusting: boolean }) {
     super();
     this.name = name;
     this.namespace = namespace;
+    this.isTrusting = isTrusting;
   }
 
   evaluate(vm: VM) {
     let { name, namespace } = this;
     let reference = vm.frame.getOperand();
-    vm.stack().setAttributeNS(namespace, name, reference);
+    vm.stack().setAttributeNS(namespace, name, reference, this.isTrusting);
```

Comment:
nit: `let { name, namespace, isTrusting } = this;` for consistency, can you fix after merge?